src/app/api/exercises/deduplicate/route.ts

```
@@ -0,0 +1,125 @@
+    })
+
+    // Store statistics for the report
+    const stats = {
+      totalExercisesProcessed: allExercises.length,
+      duplicateExerciseNamesFound: 0,
+      exercisesDeduped: 0,
+      setsUpdated: 0
+    }
+
+    // Process each group of exercises with the same name
+    for (const [name, exercises] of Object.entries(exercisesByName)) {
```

ESLint

[error] 'name' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)